spec/lib/efile/md/two_income_subtraction_worksheet_spec.rb

```diff
@@ -94,8 +120,11 @@
         intake.direct_file_json_data.interest_reports[1].recipient_tin = spouse_ssn
         intake.direct_file_json_data.interest_reports[0].amount_1099 = "1.00"
         intake.direct_file_json_data.interest_reports[1].amount_no_1099 = "2.00"
-        intake.direct_file_json_data.primary_filer&.form_1099_gs_total = "100.00"
-        intake.direct_file_json_data.spouse_filer&.form_1099_gs_total = "200.00"
+        intake.raw_direct_file_intake_data["form1099Gs"] = [{}, {}]
```

Comment:
what do all four types of income refer to?

Reply:
i ask because if unemployment is included, do we want to add data to the 1099G array? the comment says that we want to include the least amount of data to make this pass.

Reply:
so with that in mind, i may just be misunderstanding the test purpose.

Reply:
I updated the comments, hopefully it's more clear. The "minimum" refers to only populating fields used by the two income subtraction, not all the fields we expect to be present in the json. 

Also to clarify, calling `[{}, {}]` is to initialize an array json objects, and then the subsequent lines populate values. it's not the most graceful, and I'd prefer to make factory methods, but it keeps the scope low for setting up the test.

Reply:
ah, i missed the subsequent line! & that's on me, thank you for explaining.